autoPyTorch/data/base_feature_validator.py

```diff
@@ -51,7 +51,7 @@ def __init__(self,
         self.dtypes = []  # type: typing.List[str]
         self.column_order = []  # type: typing.List[str]
 
-        self.encoder = None  # type: typing.Optional[BaseEstimator]
+        self.column_transformer = None  # type: typing.Optional[BaseEstimator]
```

Comment:
So I wanted to leave these changes for your PR. Once my PR is merged, you can rebase and edit it. This will ensure we don't make the same changes twice and an easy merge.